ckanfunctionaltests/api/schemas/organization_base.schema.json

```diff
@@ -82,7 +82,6 @@
   },
   "required": [
     "id",
-    "revision_id",
```

Comment:
Also worth linking in commit 90e4fb3d790fc079285b36d84743197b1d81b9fc to show where `revision_id` was deprecated.

Reply:
Will link to changelog